workers/unity/Assets/Playground/Scripts/Cubes/ScoreCollision.cs

```diff
@@ -0,0 +1,36 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Entities;
+using UnityEngine;
+
+public class ScoreCollision : MonoBehaviour
+{
+    private EntityManager entityManager;
+
+    public void OnCollisionEnter(Collision col)
+    {
+        var component = GetComponent<SpatialOSComponent>();
+        if (!component)
+        {
+            Debug.LogError("Could not get SpatialOSComponent.");
+            return;
+        }
+        if (entityManager == null)
```

Comment:
comment seems to have disappeared, so trying it again: move lines 12 - 17 into this if-statement

Reply:
👍